easybuild/easyconfigs/m/MrBayes/MrBayes-3.2.7a-foss-2020a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'MrBayes'
+version = '3.2.7a'
```

Comment:
@lexming is there a specific reason you need this (alpha?) version?
otherwise just update it to `3.2.7` ?

Reply:
@lexming Any thoughts on the question from @smoors ?

Reply:
@smoors @verdurin version `3.2.7a` is a bugfix release, not an alpha. It dates from March 2019 while `3.2.7` dates from February 2019. There is also a note in the github release saying that version `3.2.7a` is *This should be the "latest release" (as of May 2020)*